whylog/config/__init__.py

```diff
@@ -32,6 +33,12 @@ def _load_rules(self):
     def _load_log_types(self):
         pass
 
+    def _group_parsers_by_log_type(self):
```

Comment:
Either make it a classmethod by passing parsers via argument, or pull assignment down here and rename the function (`_index_parsers()`?).